browser-app/package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "private": true,
+  "name": "browser-app",
+  "version": "0.0.1",
+  "dependencies": {
+    "@theia/core": "0.3.8",
```

Comment:
FYI: CQ for 0.3.8 hasn't been approved yet https://dev.eclipse.org/ipzilla/show_bug.cgi?id=16031